static/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -52,6 +53,10 @@ class InviteMembersModal extends DeprecatedAsyncComponent<
   InviteMembersModalProps,
   State
 > {
+  static childContextTypes = {
+    organization: SentryTypes.Organization,
+  };
```

Comment:
we typically pass the organization to the modal when its opened, is that an option?

Reply:
can we use `OrganizationContext.Provider` instead

Reply:
Thx! Good to know that we have that. I would love to use it, though the `TeamSelector` (which is widely used in the app) still uses `withOrganization` which only picks up the org from the legacy context. 🤔 
Options, I see:
1. Adjust `withOrganization` to also use `OrganizationContext `
2. Update `TeamSelector` to use the `OrganizationContext`
3. Remove it from this PR

I would remove it as 1. and 2. have a high risk of introducing bugs and / or need bigger code changes.
It should be its own PR.
Is that fine for you?

Reply:
nah just keep using what you have